boards/arm/efm32pg_stk3402a/doc/efm32wg_stk3800.rst

```diff
@@ -0,0 +1,168 @@
+.. _efm32pg_stk3402a:
+
+EFM32PG-STK3402A
+###############
+
+Overview
+********
+
+The EFM32 Pearl Gecko Starter Kit EFM32PG-STK3402A contains a MCU from the
+EFM32PG family built on ARM® Cortex®-M4F processor with excellent low
+power capabilities.
+
+Hardware
+********
+
+- Advanced Energy Monitoring provides real-time information about the energy
+  consumption of an application or prototype design.
+- Ultra low power 128x128 pixel Memory-LCD
+- 2 user buttons, 2 LEDs and a touch slider
+- Humidity, temperature, and inductive-capacitive metal sensor
+- On-board Segger J-Link USB debugger
+
+For more information about the EFM32PG SoC and EFM32PG-STK3402A board:
+
+- `EFM32PG Website`_
+- `EFM32PG Datasheet`_
```

Comment:
These links are called EFM32PG12 Datasheet (and Reference Manual) at the bottom of the doc, so you're getting a reference error from the doc generation tools.

Reply:
Done